tests/helpers/test_network.py

```diff
@@ -185,82 +177,73 @@ async def test_get_url_external(hass: HomeAssistant):
     )
 
     assert hass.config.external_url == "http://example.com:8123"
-    assert _async_get_external_url(hass) == "http://example.com:8123"
-    assert _async_get_external_url(hass, allow_cloud=False) == "http://example.com:8123"
-    assert _async_get_external_url(hass, allow_ip=False) == "http://example.com:8123"
-    assert _async_get_external_url(hass, prefer_cloud=True) == "http://example.com:8123"
+    assert _get_external_url(hass) == "http://example.com:8123"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5518609)